### PR TITLE
Vue 1087 update kv pagination usages to use new kv pager

### PR DESCRIPTION
**Pull request by dyersituations**
https://kiva.atlassian.net/browse/VUE-1087
https://kiva.atlassian.net/browse/VUE-1147
https://kiva.atlassian.net/browse/VUE-1148

- Too large offset now falls back like expected for lend API (just to 1st page)
- Browser navigation now updates page in loan channel pages
- Pager components combined and verified in all use cases

### CODE REVIEW

File: src/components/Kv/KvPagination.vue [hunk 1]
```diff
@@ -1,66 +1,67 @@
...
-		<ul class="pagination">
-			<li v-if="current !== 1" class="pagination-previous">
-				<router-link
-					:to="routeForPage(current - 1)"
-					:event="linkEventName"
-					@click.native="pageChange(current - 1, $event)"
+		<ul :class="`
+			tw-text-center
+			tw-mx-auto
+			tw-my-[.75rem]
```
Comment (mcstover) [inline src/components/Kv/KvPagination.vue:6]: Can we just use `tw-my-1.5` here? Or move to the style tag...

File: src/components/Kv/KvPagination.vue [hunk 2]
```diff
@@ -1,66 +1,67 @@
...
+			tw-flex
+			tw-justify-between
+			tw-items-center
+			tw-max-w-[17rem]`"
```
[the remark this answers is not in the document]
Comment (dyersituations) [reply src/components/Kv/KvPagination.vue]: Another old JIT class, moved to inline style.

File: src/components/Kv/KvPagination.vue (hunk 1 above)
Comment (mcstover) [reply src/components/Kv/KvPagination.vue]: No prob. thank yoU!

**Review by mcstover — APPROVED** [review]
This is a great update thanks!

**Pull request merged**